css/_details.scss

```diff
@@ -223,14 +223,30 @@ avatar.maximized .avatar-options > [class^="icon-"] {
 	padding: 20px 20px 100px;
 }
 
-@media screen and (min-width: 1450px) {
-	section {
-		-webkit-column-count: 2; /* Chrome, Safari, Opera */
-		-moz-column-count: 2; /* Firefox */
-		column-count: 2;
+/* GRID */
+$grid-height-unit: 40px;
+$grid-input-padding: 7px;
+$grid-input-margin: 3px;
+$grid-column-width: 380px;
+$grid-input-height-with-margin: calc(#{$grid-height-unit} - (#{$grid-input-margin} * 2));
```

Comment:
Calculations that are independent from the browser could be done in scss:
```scss
$grid-input-height-with-margin: #{$grid-height-unit - $grid-input-margin * 2};
```

Reply:
Nice catch! :)